terraform/environments/bootstrap/delegate-access/iam.tf

```
@@ -179,6 +179,166 @@ resource "aws_iam_policy" "member-access" {
   policy      = data.aws_iam_policy_document.member-access.json
 }
 
+module "instance-scheduler-access" {
+  count  = local.account_data.account-type == "member" && terraform.workspace != "testing-test" ? 1 : 0
```

```suggestion
  count  = local.account_data.account-type == "member" 1 : 0
```
You can create this role in the testing-test account as well so that the scheduler can also be tested in there if needed